connector-j-6/pom.xml

```diff
@@ -18,22 +18,37 @@
     </description>
 
     <dependencies>
+
         <dependency>
             <groupId>mysql</groupId>
             <artifactId>mysql-connector-java</artifactId>
             <version>6.0.6</version>
             <scope>provided</scope>
         </dependency>
+
         <dependency>
             <groupId>com.google.cloud.sql</groupId>
             <artifactId>jdbc-socket-factory-core</artifactId>
             <version>1.0.12-SNAPSHOT</version>
         </dependency>
+
         <dependency>
-            <groupId>org.eclipse.jetty</groupId>
-            <artifactId>jetty-unixsocket</artifactId>
-            <version>9.4.8.v20171121</version>
+            <groupId>com.github.jnr</groupId>
```

Comment:
Can you putt these dependencies into the parent pom so you don't have to copy them?

Reply:
Done.